mm2src/coins/tendermint/tendermint_token.rs

```diff
@@ -542,7 +550,7 @@ impl MmCoin for TendermintToken {
         Box::new(futures01::future::err(()))
     }
 
-    fn history_sync_status(&self) -> HistorySyncState { HistorySyncState::NotEnabled }
+    fn history_sync_status(&self) -> HistorySyncState { self.platform_coin.history_sync_status() }
```

Comment:
quoting from the PR's post:
> Previously it was not possible to query history of Tendermint tokens. This PR makes that
possible.

Q: is this the change that made it possible to query tendermint token history? if not, could you guide me to the change please, as there is no dedicated commit for it.

Reply:
No, not this. It's the changes made in `mm2src/coins/tendermint/tendermint_tx_history_v2.rs`.